src/nvim/ex_docmd.c

```diff
@@ -9989,23 +9989,23 @@ ArrayOf(Dictionary) get_commands(buf_T *buf)
     case(EXTRA|NEEDARG):       Arg[0] = '+'; break;
     case(EXTRA|NOSPC|NEEDARG): Arg[0] = '1'; break;
     }
-    PUT(dic, "Argument", STRING_OBJ(cstr_to_string((const char *)Arg)));
+    PUT(dic, "argument", STRING_OBJ(cstr_to_string((const char *)Arg)));
 
     // Defination
-    PUT(dic, "Defination",
+    PUT(dic, "defination",
```

Comment:
typo,  "definition" (also the comment above it)

Reply:
Okk, so, I think my english is sooo bad, I generally used defination, it was wrong, :laughing:

Reply:
And yes, I have to delete the comments?
I thought they are useful to see the code tidy.

Reply:
Today, I am laughing on myself, I didn't even know the spelling of `Definition`, how poor, haha haha :rofl:

Reply:
@nimitbhardwaj No worries!